helm/conjur-config-cluster-prep/README.md

```diff
@@ -290,6 +296,19 @@ command is run using a local copy of the Helm chart. You can use
   helm install my-conjur-release . -f my-custom-values.yaml 
   ```
 
+### Optional: Deploying the Kubernetes resources specified by this chart without Helm
+
+If Helm can not be used to deploy Kubernetes resources, the raw Kubernetes manifests can be generated ahead of time with the `helm template` command. Then the generated manifests can be applied with `kubectl`.
```

Comment:
I'm raising this purely from a standpoint of curiosity so I think it's fine to leave this as is, but I wonder what circumstances would result in both these being true

> Helm can not be used to deploy Kubernetes resources

and

> the raw Kubernetes manifests can be generated ahead of time with the `helm template` command

Reply:
I agree, it does seem strange. I can see a case where k8s admins are not permitted to use Helm to configure an actual cluster, but still use Helm on their local machines to generate manifests